recipe/run_test.py

```diff
@@ -236,138 +225,34 @@ def check_libraries(libraries_files):
    nnet3-latgen-grammar nnet3-compute-batch nnet3-latgen-faster-batch
    nnet3-latgen-faster-lookahead cuda-gpu-available cuda-compiled"""
 
-rnnlm_bins="""rnnlm-get-egs rnnlm-train rnnlm-get-sampling-lm
+rnnlm_bins = """rnnlm-get-egs rnnlm-train rnnlm-get-sampling-lm
            rnnlm-get-word-embedding rnnlm-compute-prob rnnlm-sentence-probs"""
 
-
-online_bins="""online-net-client online-server-gmm-decode-faster online-gmm-decode-faster
+if sys.platform == 'win32':
+    online_bins = """online-gmm-decode-faster
+           online-wav-gmm-decode-faster"""
+    online2_bins = """online2-wav-gmm-latgen-faster apply-cmvn-online
+         extend-wav-with-silence compress-uncompress-speex
+         online2-wav-nnet2-latgen-faster ivector-extract-online2
+         online2-wav-dump-features ivector-randomize
+         online2-wav-nnet2-am-compute  online2-wav-nnet2-latgen-threaded
+         online2-wav-nnet3-latgen-faster online2-wav-nnet3-latgen-grammar
+         online2-wav-nnet3-latgen-incremental
+         online2-wav-nnet3-wake-word-decoder-faster"""
+else:
+    online_bins = """online-net-client online-server-gmm-decode-faster online-gmm-decode-faster
            online-wav-gmm-decode-faster online-audio-server-decode-faster
            online-audio-client"""
+    online2_bins = """online2-wav-gmm-latgen-faster apply-cmvn-online
+         extend-wav-with-silence compress-uncompress-speex
+         online2-wav-nnet2-latgen-faster ivector-extract-online2
+         online2-wav-dump-features ivector-randomize
+         online2-wav-nnet2-am-compute  online2-wav-nnet2-latgen-threaded
+         online2-wav-nnet3-latgen-faster online2-wav-nnet3-latgen-grammar
+         online2-tcp-nnet3-decode-faster online2-tcp-nnet3-decode-faster-emscripten 
+         online2-wav-nnet3-latgen-incremental online2-wav-nnet3-wake-word-decoder-faster"""
```

Comment:
Could we please make it so that we have like:
```
online_bins = ...
if sys.platform != 'win32':
    # unix only, not supported on windows yet because X
    online_bins += ...
```

of, if there's a symmetric difference then 
```
online_bins = ...  # common
if sys.platform == 'win32':
    # win-only
else:
    # unix
````

Reply:
Right now it's impossible to tell (without firing up python and doing a set difference), which bins are (not) supported on which platform.